src/screens/unansweredGuidelines.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import useInitialData from 'hooks/useInitialData'
+import useInitialDataDistributor from 'hooks/useInitialDataDistributor'
+//utils
+import { getUnansweredGuidelines } from 'utils/tasks'
+//components
+import Spinner from 'components/spinner'
+import Layout from 'components/layout'
+//types
+
+const UnansweredGuidelines = (): JSX.Element => {
+  const { isLoading, error, auditProgram, distributorIds } = useInitialData()
+  const distributorId = distributorIds ? distributorIds[0] : null
+  const { isLoadingDistributor, errorDistributor, survey } = useInitialDataDistributor(
+    distributorId,
+    auditProgram
+  )
+  if (isLoading || isLoadingDistributor)
+    return (
+      <div className="mt-8 md:mt-16 mx-auto px-4">
+        <Spinner />
+      </div>
+    )
+
+  if (error || errorDistributor) return <div>Ha ocurrido un error</div>
+
+  const arrayOfUnansweredGuidelines = getUnansweredGuidelines(survey, auditProgram)
+
+  return (
+    <div>
+      <div className="text-center mt-2">
```

Comment:
Remember to add these styles to the container of the screen ```max-w-screen-sm mt-8 md:mt-16 mx-auto px-4```. It is not respecting the width and padding of the other screens